okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -592,7 +592,8 @@ private void proxyResponseImmediate(ProxyContext pc, ReadStream<Buffer> res,
         for (HttpClientRequest r : clientRequestList) {
           r.write(data);
```

Comment:
If r can handle less data per second than ctx.response() the data in r's queue will grow until out of memory.

Reply:
More theoretical than practical. But I'll change and resume when all writes are done.